infrastructures/docker/src/main/java/org/eclipse/che/workspace/infrastructure/docker/DockerInternalRuntime.java

```diff
@@ -291,13 +307,20 @@ private void startMachine(String name, DockerContainerConfig containerConfig)
                                .bootstrap();
```

Comment:
`bootstrap()` is blocking and it throws `InterruptionException` so I think that will be enough